### PR TITLE
Docs for multiple entity services

### PR DESCRIPTION
**Pull request by FrankBakkerNl**

### CODE REVIEW

File: docs/v3/hass_model/hass_model_working_with_entities.md [hunk 1]
```diff
@@ -123,12 +127,6 @@ myEntities.Sun.Sun
     });
 ```
 
-## Creating entities, setting state and updating attributes
-
-Currently, the only way to create a new entity, remove it, or set its state (outside the methods above) or set its attributes is via the [MQTT Entity Manager extension](v3/extensions/mqttEntityManager.md).
```
Comment (skotl) [inline docs/v3/hass_model/hass_model_working_with_entities.md:128]: Is it not worth leaving in a signpost to the extension for anyone who wants to create entities or set attributes (I know that state is probably easier via the ha services)?
I know I struggled for some time trying to understand whether it was possible to create entities and set their attributes and if we remove the mention here then people will literally have to read all the way down to the final page of the docs to discover this functionality.

**Comment by helto4real** [comment]
Looks like CI failed

File: docs/v3/hass_model/hass_model_working_with_entities.md (hunk 1 above)
Comment (FrankBakkerNl) [reply docs/v3/hass_model/hass_model_working_with_entities.md:128]: This was caused by merging main back to my branch. So it was not intended. I do however doubt if it really should be here,.I saw there was also such a mention in the getting started example, I think it is a bit much to mention this extension in all these places.
@helto4real ?
I did make it more explicit that these Entity classes are wrappers for existing entities and do not create new ones.

Comment (FrankBakkerNl) [reply docs/v3/hass_model/hass_model_working_with_entities.md:128]: I did also start on an overview page that explains the goal of the core components, Appmodel HassModel etc. Maybe we should also list the extensions there so a user gets an overview of them all without having the docs to cross reference each other to much

Comment (skotl) [reply docs/v3/hass_model/hass_model_working_with_entities.md:128]: That sounds a great idea - thanks. Let me know if I can help.

Comment (helto4real) [reply docs/v3/hass_model/hass_model_working_with_entities.md:128]: Sounds great. An overview would help a lot indeed rather than reference it everywhere.

**Review by helto4real — APPROVED** [review]

**Pull request merged**